index.html

```diff
@@ -2133,8 +1145,8 @@ <h2>Resource Integrity</h2>
 property named <dfn class="lint-ignore">`digestMultibase`</dfn> in any object
 that includes an `id` property. If present, the `digestMultibase` value MUST be
 a single string value, or an array of string values that are
```

Comment:
travels with the change below
```suggestion
a single string value, or an array of string values, each of which is a
```